notebooks/train_with_additional_metadata/train_ms2deepscore/old_run_scripts/calculate_tanimoto_scores.py

```diff
@@ -0,0 +1,15 @@
+import os
```

Comment:
Below there are many run functions that were used to create models. I will remove these, since they are not necessary to share on github.